tm/src/main/java/com/alibaba/fescar/tm/api/transaction/TransactionHookManager.java

```diff
@@ -26,7 +26,7 @@
  */
 public final class TransactionHookManager {
 
-    private static final ThreadLocal<List<TransactionHook>> hooksLocal=new ThreadLocal<>();
+    private static final ThreadLocal<List<TransactionHook>> hooksLocal = new ThreadLocal<>();
```

Comment:
Let's call a static final member as uppercase:
```java
private static final ThreadLocal<List<TransactionHook>> LOCAL_HOOKS = new ThreadLocal<>();
```

Reply:
why not use spi to load the transaction hook instead of a thread local holder?